src/html/layouts/application.html

```diff
@@ -13,7 +13,17 @@
     {% block content %}{% endblock %}
 
     {% block javascript %}
+      <script src="https://polyfill.io/v2/polyfill.min.js?features=IntersectionObserver"></script>
```

Comment:
Seems like you have 3 different polyfills going on here:
1. the polyfill from polyfill.io
2. the polyfill specified in package.json (but not imported anywhere)
3. the polyfill in src/javascripts/intersection-observer/intersection-observer.js

Unless I'm missing something, you should only need one of these.